packages/tabs/src/Tabs/Tabs.types.ts

```diff
@@ -1,57 +1,67 @@
-import { Either, HTMLElementProps, LgIdProps } from '@leafygreen-ui/lib';
+import {
+  DarkModeProps,
+  Either,
+  HTMLElementProps,
+  LgIdProps,
+} from '@leafygreen-ui/lib';
+import { PolymorphicAs } from '@leafygreen-ui/polymorphic';
 import { BaseFontSize } from '@leafygreen-ui/tokens';
 
-export interface TabsProps extends HTMLElementProps<'div'>, LgIdProps {
+export interface TabsProps
+  extends HTMLElementProps<'div'>,
+    DarkModeProps,
+    LgIdProps {
   /**
-   * Content that will appear inside of Tabs component.
-   * Should be comprised of at least two `<Tab />` components.
+   * Accessible label that describes the set of tabs
    */
-  children: React.ReactNode;
+  ['aria-label']?: string;
 
   /**
-   * Content that will appear inline after the `<Tab />` components. `inlineChildren` are wrapped in a flexbox container.
+   * References id of label external to the component that describes the set of tabs
    */
-  inlineChildren?: React.ReactNode;
+  ['aria-labelledby']?: string;
 
   /**
-   * Callback to be executed when Tab is selected. Receives index of activated Tab as the first argument.
-   *
-   * @type (index: number) => void
+   * HTML Element that wraps title in Tab List.
    */
-  setSelected?: React.Dispatch<number>;
+  as?: PolymorphicAs;
 
   /**
-   * Index of the Tab that should appear active. If value passed to selected prop, component will be controlled by consumer.
+   * The base font size of the title and text rendered in children.
    */
-  selected?: number;
+  baseFontSize?: BaseFontSize;
 
   /**
-   * determines if component will appear for Dark Mode
-   * @default false
+   * Content that will appear inside of Tabs component.
+   * Should be comprised of at least two `<Tab />` components.
    */
-  darkMode?: boolean;
+  children: React.ReactNode;
 
   /**
-   * HTML Element that wraps title in Tab List.
+   * When set to true, all tab panels will forcibly render in DOM.
+   * Tab panels that are not selected will be hidden with `display: none;`
+   * This will not apply for disabled tabs
    *
-   * @type HTMLElement | React.Component
+   * @default false
    */
-  as?: React.ElementType<any>;
+  forceRenderAllTabPanels?: boolean;
```

Comment:
(I realize you've probably got approval on this already, so this is a big nit): do we need the word "force" here? Does the prop name make semantic sense as just `renderAllPanels`?

Reply:
I added the `force*` because we wanted to convey some element of this being an exception to the norm